WordPress/Classes/Models/AbstractPost.swift

```diff
@@ -1,10 +1,81 @@
 import Foundation
 
 extension AbstractPost {
+
+    // MARK: - Status
+
+    @objc
+    var statusTitle: String? {
+        guard let status = self.status else {
+            return nil
+        }
+
+        return AbstractPost.title(for: status)
+    }
+
+    @objc
+    var remoteStatus: AbstractPostRemoteStatus {
+        get {
+            guard let remoteStatusNumber = remoteStatusNumber?.uintValue,
+                let status = AbstractPostRemoteStatus(rawValue: remoteStatusNumber) else {
+                    return .pushing
```

Comment:
This is an unfortunate Swift conversion because we are required to provide a default value. But `.pushing` seems to be an incorrect default value for me. 

I do not think we should fix this though. My only concern here is, for someone new looking at this code, I don't quite understand the reason **why** `.pushing` is returned in the case that `remoteStatusNumber` is invalid. 

The `remoteStatusNumber` definition in the Core Data DB is also weird in that it has a default value, `0`, but is optional. ¯\\\_(ツ)\_/¯

Reply:
> This is an unfortunate Swift conversion because we are required to provide a default value. But .pushing seems to be an incorrect default value for me.
> 
> I do not think we should fix this though. My only concern here is, for someone new looking at this code, I don't quite understand the reason why .pushing is returned in the case that remoteStatusNumber is invalid.
> 
> The remoteStatusNumber definition in the Core Data DB is also weird in that it has a default value, 0, but is optional. ¯\_(ツ)_/¯

I agree.  And I also agree we shouldn't fix this for this PR, although I'd like to explain a bit more about "why".

The intent of this PR was to offer a migration that accurately maintains the existing logic.  This will allows us in part to minimize the intrinsic risk of the migration, and in part to scope what the extent of my work was for this PR making the review hopefully easier.

In a way, this migration is good because it highlights a shortcoming in the previously-implicit default value for this property that was sort of hidden before now.